### PR TITLE
Default path for MRIUploadIncomingPath

### PR DESCRIPTION
**Pull request by MounaSafiHarab**
Set default to /data/incoming (like what we have in the installation scripts) to prevent imaging_uploader from uploading to /tmp/

### CODE REVIEW

File: SQL/2016-03-10-DefaultMRIUploadPath.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+UPDATE Config AS c, ConfigSettings AS cs SET c.value="/data/incoming/" WHERE c.ConfigID=cs.ID AND cs.Name="MRIUploadIncomingPath";
```
Comment (driusan) [inline SQL/2016-03-10-DefaultMRIUploadPath.sql]: This should include add a where clause to only update if it's set to the default, and not overwrite existing variables. (Patch only)

**Comment by xlecours** [comment]
Should the existing projects apply this patch?

**Comment by driusan** [comment]
I don't think it matters.. if they've customized it, the WHERE clause will make it a no-op.

**Comment by christinerogers** [comment]
@MounaSafiHarab tagged for existing projects.  adding to notes

**Pull request merged**